### ISSUE TITLE
[Request/Compat] Pumping water amd lava in and out of caldrons 

### ISSUE THREAD

**Issue by AbacusOperator** [ItsBlackGear/CavesAndCliffsBackport#11]
Many automations mods allow pumping fluids and storing them in tanks. It would be nice if it was possible to automate caldrons (especially lava production). For example, Flopper mod allows movement fluid analogous to that of hopper for items.

**Comment by ItsBlackGear** [comment]
does it support other modded fluid containers? and if it does please give examples

**Comment by AbacusOperator** [comment]
Flopper that I used as example for pumping is compatible with all technological mods with fluid tanks that I am aware of: Mekanism, Create, Industrial Foregoing, Tinker's Construct, etc. These mods also have their own pipes for fluid transportation. So it would be very nice if you re-coded caldrons as 1000mb tanks that only admit vanilla water and lava as their content. It would make possible to automate 1.17's dripstone lava production.

**Comment by LatiusAuro** [comment]
Yeah, this is something I would want to see. Although I take it that part of the issue would be how to do the water stuff with them - normally, cauldrons have three water states: each equivalent to 1 bottle of water in between them - so it might not be actually possible as such.

(Although I think Create does have a way which could work - you'd need a deployer filtered to use buckets, and some brass funnels, belts and a way to extract the lava from the buckets. It may not always be an option though, especially not for large scale lava farms, so an easier way would also help.)

**Comment by TrkvonStpd1e11** [comment]
Yeah. Create 0.4 in 1.18 can extract lava from lava cauldrons using fluid components.
